libcalico-go/lib/apis/v3/node.go

```diff
@@ -56,10 +56,10 @@ type NodeSpec struct {
 	VXLANTunnelMACAddr string `json:"vxlanTunnelMACAddr,omitempty" validate:"omitempty,mac"`
 
 	// IPv6VXLANTunnelAddr is the address of the IPv6 VXLAN tunnel.
-	IPv6VXLANTunnelAddr string `json:"ipv6VXLANTunnelAddr,omitempty" validate:"omitempty,ipv4"`
+	IPv6VXLANTunnelAddr string `json:"ipv6VXLANTunnelAddr,omitempty" validate:"omitempty,ipv6"`
 
-	// IPv6VXLANTunnelMACAddr is the MAC address of the IPv6 VXLAN tunnel.
-	IPv6VXLANTunnelMACAddr string `json:"ipv6VXLANTunnelMACAddr,omitempty" validate:"omitempty,mac"`
+	// VXLANTunnelMACAddrV6 is the MAC address of the IPv6 VXLAN tunnel.
+	VXLANTunnelMACAddrV6 string `json:"vxlanTunnelMACAddrV6,omitempty" validate:"omitempty,mac"`
```

Comment:
Just curious, why the rename?

Reply:
Oh this was addressing a comment by @mgleung on a previous PR that I merged without addressing, so I snuck it in here (the original reasoning is that most of our variable names have "V6" at the end, but the existing IPv4VXLANTunnelAddr broke that pattern, so I changed the MAC tunnel addr to use this pattern IPv4VXLANTunnelAddr/IPv6VXLANTunnelAddr being the only ones that break it)

Reply:
Works for me. I think we're not super consistent here sadly (see `IPv4VXLANTunnelAddr`) but given VXLANTunnelMACAddr doesn't have an `IPv4` prefix, I suppose using the `v6` suffix is the way to be self-consistent here.

Reply:
Thanks!